### PR TITLE
docs: Add weekly-report (0814-0827)

### PR DESCRIPTION
**Pull request by saltism**
weekly-report (0814-0827)

### CODE REVIEW

File: blog/en/blog/2023/08/30/weekly-report.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+- Apache APISIX
+- API Gateway
+- Weekly Report
+- Contributor
+description: The Apache APISIX Community Biweekly Report can help community members better grasp the progress of the Apache APISIX community so that everyone can participate in the Apache APISIX community.
+tags: [Community]
+image: https://static.apiseven.com/uploads/2023/08/28/riVgERes_%E8%8B%B1%E6%96%87%E5%A4%B4%E5%9B%BE.png
+---
+
+> We have recently made fixes and improvements to certain features of Apache APISIX. These include adding proxy configuration item for `openid-connect` plugin, fixing DNS resolution issue for tencent-cloud-cls, adding pass host configuration capability for ApisixUpstream resource, upgrading Go toolchain to v1.20, etc. For more details, please refer to the biweekly report.
```
Comment (Yilialinn) [inline blog/en/blog/2023/08/30/weekly-report.md]:
```suggestion
> We have recently made fixes and improvements to certain features of Apache APISIX and APISIX Ingress Controller. These include adding proxy configuration item for `openid-connect` plugin, fixing DNS resolution issue for tencent-cloud-cls, adding pass host configuration capability for ApisixUpstream resource, upgrading Go toolchain to v1.20, etc. For more details, please refer to the biweekly report.
```

File: blog/en/blog/2023/08/30/weekly-report.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+### Apache APISIX
+
+- [Add proxy configuration item for `openid-connect` plugin](https://github.com/apache/apisix/pull/9948) (Contributor: [darkSheep404](https://github.com/darkSheep404))
+
+- [Fix DNS resolution issue for tencent-cloud-cls](https://github.com/apache/apisix/pull/9843) (Contributor: [jiangfucheng]((https://github.com/jiangfucheng))
+
+- [Admin-API supports api schema validation](https://github.com/apache/apisix/pull/10065) (Contributor: [kingluo](https://github.com/kingluo))
+
+- [`WASM` plugin supports passing native JSON data structure](https://github.com/apache/apisix/pull/10072) (Contributor: [Sn0rt](https://github.com/Sn0rt))
+
```
Comment (Yilialinn) [inline blog/en/blog/2023/08/30/weekly-report.md:56]:
```suggestion

### APISIX Ingress Controller

```

**Review by Yilialinn — COMMENTED** [review]
Revise the Chinese version accordingly @JYan00

**Comment by saltism** [comment]
> Revise the Chinese version accordingly @JYan00

Done.

**Review by Yilialinn — APPROVED** [review]

**Review by yzeng25 — APPROVED** [review]
LGTM

**Review by guoqqqi — APPROVED** [review]

**Pull request merged**